css-viewport/Overview.bs

```diff
@@ -356,6 +356,154 @@ only the value previously set to it.
 }
 </pre>
 
+<h2 id='zoom-property'>
+The 'zoom' property</h2>
+
+An element becomes zoomed when the 'zoom' property has a positive computed value
```

Comment:
I would suggest adopting https://rhodesmill.org/brandon/2012/one-sentence-per-line/ and maybe also consider https://lists.w3.org/Archives/Public/spec-prod/2018OctDec/0011.html (last section)